src/modal/modal.js

```diff
@@ -555,20 +557,65 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap', 'ui.bootstrap.p
 
         openedWindows.top().value.modalDomEl = angularDomEl;
         openedWindows.top().value.modalOpener = modalOpener;
+
+        applyAriaHidden(angularDomEl[0]);
```

Comment:
I think the subsequent code would be nicer if you just passed `angularDomEl`, and access the raw element only when needed, i.e `el[0].tagName`. This would be more in keeping with usage of jqLite whenever possible.